src/feedinlib/era5.py

```diff
@@ -322,9 +342,11 @@ def select_geometry(ds, area):
         logical_list.append(
             np.logical_and((ds.longitude == lon), (ds.latitude == lat))
         )
-
```

Comment:
The coordinates are fine here: lon=13.1 and lat=52.3

But the logical_list got the True at the wrong position (row:0, column:2) instead of (row:2, column:0).
```python
[<xarray.DataArray 'longitude' (longitude: 3, latitude: 3)>
array([[False, False,  True],
       [False, False, False],
       [False, False, False]])
Coordinates:
  * longitude  (longitude) float32 13.1 13.35 13.6
  * latitude   (latitude) float32 52.8 52.55 52.3]
```

I have no idea why this happens.  @oemof/feedinlib Any ideas?